packages/tools/src/groupFilesByType.ts

```diff
@@ -4,22 +4,11 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @flow
  */
 
-import {groupBy} from 'lodash';
+import { groupBy } from 'lodash';
```

Comment:
fix spacing

Reply:
I will need to configure linter with prettier for Typescript.